test/jdk/javax/swing/JComboBox/bug4436376.java

```diff
@@ -41,8 +41,9 @@
 public class bug4436376 {
     static JFrame frame;
     static volatile Point p;
+    static volatile JComboBox combo;
```

Comment:
Making Swing object volatile does not mean that calling its methods not from EDT is Ok. As a solution you can perform the entire check block together with throwing an exception inside the invokeAndWait block. Same with the second test.

Reply:
done